src/client/v2/algod/models/types.ts

```diff
@@ -1685,8 +1685,8 @@ export class TransactionParametersResponse extends BaseModel {
   /**
```

Comment:
Could you remove the changes to this file since it's generated from the algod API spec? If these are included, I think we'll get an autogenerated PR to change this file back.

We should change the descriptions for fees in algod instead.